internal/consensus/state.go

```diff
@@ -1407,10 +1444,12 @@ func (cs *State) defaultDoPrevote(height int64, round int32) {
 		}
 
 		if cs.Proposal.POLRound == -1 {
-			logger.Debug("prevote step: Proposal is timely",
-				"timestamp", cs.Proposal.Timestamp.Format(time.RFC3339Nano),
-				"receive_time", cs.ProposalReceiveTime.Format(time.RFC3339Nano),
-				"timestamp_difference", cs.ProposalReceiveTime.Sub(cs.Proposal.Timestamp))
+			if logger.DebugOn() {
+				logger.Debug("prevote step: Proposal is timely",
+					"timestamp", cs.Proposal.Timestamp.Format(time.RFC3339Nano),
```

Comment:
This is another case in which makes sense, as we expend time formatting timestamps.